package.json

```diff
@@ -16,6 +16,8 @@
     "build-docs": "yarn cmsds build-docs",
     "build-docs:healthcare": "yarn cmsds build-docs --config cmsds.healthcare.config.js",
     "build-docs:medicare": "yarn cmsds build-docs --config cmsds.medicare.config.js",
+    "build:gatsby": "yarn --cwd ./packages/docs build",
```

Comment:
maybe build:docs, start:docs ?

Reply:
we can change this when we retire the legacy one

Reply:
I struggled with naming here because I didn't want to conflict with the current doc site. 
And to Patrick's point, I think maybe this can be updated right before we merge this feature work?

Reply:
Yeah no biggie, object:verb or verb:object! We use a little of both.. always something we can update in the future..